homeassistant/components/transmission/__init__.py

```diff
@@ -210,12 +214,39 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     return unload_ok
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate an old config entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    # Version 1 -> 2:
+    #   - Adding ssl
+    #   - Adding path
+    if config_entry.version == 1:
+        new = {**config_entry.data}
+
+        # add ssl
+        if CONF_SSL not in new:
```

Comment:
If config version is 1 then definitely these keys won't be in `data`

Reply:
So you would suggest by removing that check altogether  because it is practically meaningless?